content/en/serverless/azure_app_services.md

```diff
@@ -130,25 +130,107 @@ DogStatsd.Increment("sample.startup");
+
+### Updating the extension for a resource group {#powershell-resource-group}
+
+The update script applies to an entire resource group. This script updates every instance (Web App or Function App) that has the extension installed. App Service instances that do not have the Datadog extension installed are not affected.
+
+1. Open the Azure CLI or Azure Cloud Shell.
+2. Download the installation script using the following command:
+
+    ```
+    $baseUri="https://raw.githubusercontent.com/DataDog/datadog-aas-extension/master/management-scripts/extension"; Invoke-WebRequest -Uri "$baseUri/update-all-site-extensions.ps1" -OutFile "update-all-site-extensions.ps1"; Invoke-WebRequest -Uri "$baseUri/install-latest-extension.ps1" -OutFile "install-latest-extension.ps1"
+    ```
+
+3. Run the following command. All arguments are required.
+
+    ```
+    .\update-all-site-extensions.ps1 -SubscriptionId <SUBSCRIPTION_ID> -ResourceGroup <RESOURCE_GROUP_NAME> -Username <USERNAME> -Password <PASSWORD>
+    ```
+
+### Installing a specific version of the extension {#powershell-specific-version}
+
+The Azure App Service UI does not support the ability to install a specific version of an extension. You may do this with the install or update script.
+
+#### Install specific version on a single resource
+
+To install a specific version on a single instance, follow the [instructions for installing the extension for the first time](#powershell-first-time) and add the `-ExtensionVersion` parameter to the installation command.
+
+```
+.\install-latest-extension.ps1 -Username <USERNAME> -Password <PASSWORD> -SubscriptionId <SUBSCRIPTION_ID> -ResourceGroup <RESOURCE_GROUP_NAME> -SiteName <SITE_NAME> -DDApiKey <DATADOG_API_KEY> -ExtensionVersion <EXTENSION_VERSION>
+```
+
+Replace `<EXTENSION_VERSION>` with the version of the extension you wish to install. For instance, `1.4.0`.
+
+#### Install specific version on an entire resource group
+
+To install a specific version for a resource group, follow the [instructions for updating the extension for a resource group](#powershell-resource-group) and add the `-ExtensionVersion` parameter to the installation command.
+
+```
+.\update-all-site-extensions.ps1 -SubscriptionId <SUBSCRIPTION_ID> -ResourceGroup <RESOURCE_GROUP_NAME> -Username <USERNAME> -Password <PASSWORD> -ExtensionVersion <EXTENSION_VERSION>
+```
+
+Replace `<EXTENSION_VERSION>` with the version of the extension you wish to install. For instance, `1.4.0`.
+
 ## Troubleshooting
 
-1. If you are missing metrics and metadata in the APM trace panel and service page:
+### If you are missing metrics and metadata in the APM trace panel and service page
 
 It is likely that you do not have the Azure integration configured to monitor your application. Proper configuration improves your ability to correlate metrics, traces, and logs in the Datadog platform, as well as to avoid broken or empty screens. To fix this:
```

Comment:
```suggestion
It is likely that you do not have the Azure integration configured to monitor your application. Proper configuration improves your ability to correlate metrics, traces, and logs on the Datadog site, as well as to avoid broken or empty screens. To fix this:
```